client/src/app/components/Cart/ItemInCart.js

```diff
@@ -0,0 +1,154 @@
+import React, { Component } from 'react';
+import { Notify } from 'react-redux-notify';
+import NotifyService from '../../../utils/notify.service';
+import styles from './Cart.module.scss';
+import { productType } from '../../types';
+
+class ItemInCart extends Component {
+  static propTypes = {
+    item: productType
+  };
+
+  static defaultProps = { item: null };
+
+  state = { sizeClicked: '', activeColor: '', quantity: 1 };
+
+  increment = () => {
+    const { quantity } = this.state;
+    const { createNotification, setQuantityAndTotal, item } = this.props;
+    const { _id } = item;
+    const newQuantity = quantity + 1;
```

Comment:
probably it could be redundant
if the quantity is stored in redux it could be passed through props, isn't?

Reply:
changed